README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
+
+Install dependencies
+
+```sh
```

Comment:
After the command for running the server in watch mode, suggest a command to stop the server, like `Ctrl+C`, to inform users about how to exit that mode.